inspire/dojson/processors.py

```diff
@@ -17,11 +17,29 @@
 # along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
+"""Convert incoming MARCXML to JSON."""
+
 
 def convert_marcxml(source):
     """Convert MARC XML to JSON."""
-    from inspire.dojson import inspiremarc
     from dojson.contrib.marc21.utils import create_record, split_blob
+    from inspire.dojson.hep import hep
+    from inspire.dojson.institutions import institutions
 
     for data in split_blob(source.read()):
-        yield inspiremarc.do(create_record(data))
+        record = create_record(data)
+        if _collection_in_record(record, 'institution'):
```

Comment:
So, this is like parameters to the functions specified in the `RECORD_PROCESSORS` config (suggested by Javier). Since `RECORD_PROCESSORS` is already meant to be overridden on the overlay, this `CFG_DOJSON_MAPPINGS ` could live in the overlay only and not needed to be upstream. And then it could even just be hard-coded in the function we have on the overlay since it is a customisation anyway, if that make sense. 

On the other hand, we could suggest a patch to `invenio-records` to make the default `convert_records` use `CFG_DOJSON_MAPPINGS` config. But I suggest we merge this as is now and see when `invenio-records` matures.